.azure-pipelines-templates/matrix.yml

```diff
@@ -21,7 +21,7 @@ parameters:
     SGX:
       cmake_args: "-DCOMPILE_TARGETS=sgx"
     debug:
-      cmake_args: "-DCMAKE_BUILD_TYPE=Debug -DLVI_MITIGATIONS=OFF"
+      cmake_args: "-DCMAKE_BUILD_TYPE=Debug -DLVI_MITIGATIONS=OFF -DVERBOSE_LOGGING=ON"
```

Comment:
I wonder what the impact of this is on the CI pipeline duration/size of artefacts published on error?